### PR TITLE
MAIN - move to yarn and bump dependencies

### PR DESCRIPTION
**Pull request by Weetbix**
This PR: 
- Moves the project to `yarn`
- Uses the `resolutions` feature of `yarn` to override the dependency versions of braces and lodash in order to use versions without security alerts
  - https://yarnpkg.com/lang/en/docs/selective-version-resolutions/
  - karma-webpack minimum version is bumped to the highest minor version in order that lodash can be upgraded.

### CODE REVIEW

**Comment by Weetbix** [comment]
@paschalidi I will update most places to use yarn, but I will keep the publish and versioning commands using npm. 

npm is bundled with node so there's no chance to not have it, and I don't want to update these commands for no reason as it will add complexity

Sound good?

**Comment by paschalidi** [comment]
yeah makes sense to me, thanks for the changes

File: package.json [hunk 1]
```diff
@@ -45,14 +45,18 @@
...
     "rimraf": "^2.6.3",
     "webpack": "^4.35.2"
   },
+  "resolutions": {
+    "karmatic/karma-webpack": "^2.0.13",
+    "karmatic/karma/expand-braces/braces": "^2.3.2"
+  },
   "husky": {
     "hooks": {
       "pre-commit": "npm run linter:check && npm test && pretty-quick --staged"
```
Comment (paschalidi) [inline package.json]: hm, maybe you forgot this?

**Comment by paschalidi** [comment]
Checked locally for `npm` instances in text, maybe can you have a look also [here](https://github.com/VivyTeam/krypt-web/blob/master/CONTRIBUTING.md),

**Comment by Weetbix** [comment]
Oh yeah :)

**Pull request merged**